<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brick Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { background-color: #282c34; }
QLabel { color: #FFFFFF; }
QPushButton {
        background-color: #61afef;
        color: #282c34;
        border: none;
        border-radius: 10px;
        padding: 10px;
        font-size: 16px;
}
QPushButton:hover {
        background-color: #528bde;
}
QPushButton:pressed {
        background-color: #3a6cb1;
}</string>
  </property>
  <widget class="QWidget" name="QW_Central">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="QVBL_menu" stretch="0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="QL_Title">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Brick Game</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Start">
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Tetris">
        <property name="text">
         <string>TETRIS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Snake">
        <property name="text">
         <string>SNAKE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Exit">
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="QL_MenuImage">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../resource.qrc">:/im/images/turtle.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
